### DISCUSSION TITLE
SVG images cannot be displayed

### DISCUSSION THREAD

**Discussion by nmnmcc** [unoplatform/uno#17511] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/a942c71a-6a60-458d-a7c2-1b03f76bd97b)

I have tried the following paths:

- Assets/copyleft.svg
- ms-appx:///Assets/copyleft.svg
- ms-appx:///[AppName]/Assets/copyleft.svg
- ms-appx://[AppName]/Assets/copyleft.svg

Only using absolute paths can make SVG display normally.

PNG images can be displayed normally using ms-appx:///Assets/copyleft.svg.png.

I have tried using WinUI 3 and Skia, both have the same problem.

**Comment by morning4coffe-dev** [comment, marked as answer]
Hey @intzaaa, when you add an SVG to your project, it is automatically converted via our Resizetizer for best performance. Using the PNG is the recommended way for consistent image rendering across all platforms. You should be able to use it with just `ms-appx:///Assets/copyleft.png`.

If you still want to display the SVG image directly, you can set the build action of the SVG file to `Content`, and you should be able to reference the SVG. Here are our docs for more details: https://platform.uno/docs/articles/external/uno.resizetizer/doc/using-uno-resizetizer.html?tabs=singleproject%2CAndroid#using-svg-images-vs-png-images-with-svg-underneath

**Reply by nmnmcc** [reply]
Thanks!
